tests/detectors/test_glasses_detector.py

```diff
@@ -9,38 +9,45 @@
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
 from os.path import abspath
-import mock
 
+import mock
 from preggy import expect
-from tests.base import DetectorTestCase
+from tornado.testing import gen_test
 
+from tests.base import DetectorTestCase
 from thumbor.detectors.glasses_detector import Detector as GlassesDetector
 
 
 class GlassesDetectorTestCase(DetectorTestCase):
-    def test_detector_uses_proper_cascade(self):
-        cascade = './tests/fixtures/haarcascade_eye_tree_eyeglasses.xml'
+    @gen_test
+    async def test_detector_uses_proper_cascade(self):
+        cascade = "./tests/fixtures/haarcascade_eye_tree_eyeglasses.xml"
         ctx = mock.Mock(
-            config=mock.Mock(
-                GLASSES_DETECTOR_CASCADE_FILE=abspath(cascade),
-            )
+            config=mock.Mock(GLASSES_DETECTOR_CASCADE_FILE=abspath(cascade),)
         )
 
         detector = GlassesDetector(ctx, 1, [])
         expect(detector).not_to_be_null()
 
-    def test_should_detect_glasses(self):
-        with open(abspath('./tests/fixtures/images/Christophe_Henner_-_June_2016.jpg'), 'rb') as f:
+    @gen_test
+    async def test_should_detect_glasses(self):
+        with open(
+            abspath(
+                "./tests/fixtures/images/Christophe_Henner_-_June_2016.jpg"
+            ),
+            "rb",
+        ) as f:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "f" doesn't conform to snake_case naming style](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)